### ISSUE TITLE
Single window mode

### ISSUE THREAD

**Issue by lkjell** [hensm/disable_tab_detach#1]
Could you make it single window mode too?

**Comment by hensm** [comment]
How do you mean? Redirect tabs from newly opened windows to the existing window?

**Comment by lkjell** [comment]
Yes. Redirect all tabs to one window. New windows will be in existing window.

**Comment by hensm** [comment]
I suppose it's related in function, but I think it'd be better as a separate extension. I prefer not to add options pages to simple extensions where the options page ends up being the most complex part.

I'll look at this later.
